libs/dextool/source/dextool/compilation_db/package.d

```diff
@@ -701,6 +703,17 @@ ParseFlags parseFlag(CompileCommand cmd, const CompileCommandFilter flag_filter,
         return 0 != flag.among("-D");
     }
 
+    /// Flags that is includes but contains spaces are wrapped in quotation marks.
+    static bool isSpacedInclude(string flag) @safe {
```

Comment:
I think you need to do something like `isCombinedIncludeFlag` and `isFlagAndPath` because both these are valid arguments that we need to detect:
`-I"foo/bar"` and `-I "foo/bar`. Notice the space.... yes there exists build systems in the world that do it that way...